test/ensureArray.js

```diff
@@ -0,0 +1,31 @@
+import { assert } from 'chai';
+import * as RA from '../src/index';
```

Comment:
insert one empty line between absolute and relative imports. Prettier seems to not modify this.